fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1735,4 +1735,38 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         key = appContext.getPreferenceKey(R.string.pref_key_enable_tabs_tray_to_compose),
         default = FeatureFlags.composeTabsTray,
     )
+
+    /**
+     * Adjust Activated User sent
+     */
+    var adjustUserActivatedSent by booleanPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_adjust_user_activated_sent),
+        default = false,
+    )
+
+    /**
+     * Indicates how many days in the first week user opened the app.
+     */
+    val adjustUseCount = counterPreference(
+        appContext.getPreferenceKey(R.string.pref_key_adjust_early_browse_count),
+        maxCount = 2,
+    )
+
+    var adjustUseCountLastIncrement by longPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_adjust_early_browse_count_last_increment),
+        default = 0L,
+    )
+
+    /**
+     * Indicates how many days in the first week user searched in the app.
+     */
+    val adjustEarlySearchCount = counterPreference(
+        appContext.getPreferenceKey(R.string.pref_key_adjust_early_search),
+        maxCount = 1,
+    )
+
+    var adjustEarlySearchCountLastIncrement by longPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_adjust_early_search_count_last_increment),
+        default = 0L,
+    )
```

Comment:
I'm not sure if this is still the case, but I was cautioned to use `GrowthData` language instead of `Adjust` language when this project first began, which is why the existing pref names are following that convention. I don't think I care enough to block the PR on it, but it might help with consistency to follow the existing naming if you're so inclined.

Reply:
gotcha, I'll make the adjustment (*buh-dum-tss*)